lib/tracking_options.dart

```diff
@@ -0,0 +1,114 @@
+import 'constants.dart';
+
+/// Disable tracking of user properties.
+///
+/// Before initialzing Amplitude instance, create a TrackingOptions instance
+/// with your desired tracking options and pass it to the Amplitude instance.
+class TrackingOptions {
+  Set<String> _disabledFields = {};
+
+  // Available on iOS, Android, and Web
+  TrackingOptions diableIpAddress() {
```

Comment:
Ther is a spelling mistake in all of these functions - should be "disableIpAddress" etc. not "diable..."

Reply:
Thanks @john-paradym, it's been fixed https://github.com/amplitude/Amplitude-Flutter/pull/162/commits/66e51d50549b36838d010e46a9b2a0c20ef9cfa0